cli/src/commands/auth.rs

```diff
@@ -67,7 +91,7 @@ pub async fn handle_auth(
             }
         }
     } else if matches.subcommand_matches("status").is_some() {
-        handle_auth_status(&config);
+        handle_auth_status(config, timeout, ignore_certs).await?;
```

Comment:
Currently only error values from `handle_auth_status` are propagated, which should be fine since it's otherwise always `Void` anyway. But generally I think it would make sense to have each of these branches return their function's value, instead of having a `CommandValue::Void.into()` returned at the end.

I've decided against refactoring this to keep this PR minimal.

Reply:
I agree. That refactor is needed, but that can be a separate PR.